CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased [(diff)][diff-unreleased]
 
-## [0.2.0] - 2020-11-19 - [(diff with 0.1.0)][diff-0.2.0]
+## [0.2.0] - 2020-11-19 - [(diff with 0.1.0)](https://github.com/pubgrub-rs/pubgrub-rs/compare/v0.1.0...v0.2.0)
```

Comment:
```suggestion
## [0.2.0] - 2020-11-19 - [(diff with 0.1.0)](https://github.com/pubgrub-rs/pubgrub/compare/v0.1.0...v0.2.0)
```

Weird that github manages to resolve the link even when it is wrong.
Is there a particular reason you prefer those been inlined? (I'm fine with it)

Reply:
> Weird that github manages to resolve the link even when it is wrong.

Well, this repo was named `pubgrub-rs` and we renamed it after transferring to the organization, so there is a redirect established.
It was not my intention to use it though, thanks!

As for inlining... I didn't think that having a reusable link to the diff serves any purpose as it's used only in one place. Also when the amount of diffs grows with new releases, it would grow the bottom of the file.

If you think it makes it easier to add new links in the future, I could revert the inlining.